Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -215,8 +215,12 @@ private static void Init()
         /// </remarks>
         public static void RefreshFolders()
         {
-            string path = Application.dataPath;
-            searchForFoldersTask = Task.Run(() => SearchForFoldersAsync(path));
+            // MRTK may be located in Assets (.unitypackage import) or the Packages (UPM import)
+            // folder. Check both locations.
+            List<string> rootFolders = new List<string>();
+            rootFolders.Add(Application.dataPath);
+            rootFolders.Add(Path.GetFullPath("Packages"));
+            searchForFoldersTask = Task.Run(() => SearchForFoldersAsync(rootFolders));
```

Comment:
In my testing, I found one more set of folders we need to search, the package cache. I think you need to add something like:

```
rootFolders.Add(Path.GetFullPath(Path.Combine("Library", "PackageCache")));
```

Reply:
@davidkline-ms I think this another data point for using the `AssetDatabase` APIs instead of the way the MixedRealityToolkitFiles class is currently written. Those APIs _should_ search across all Assets and Packages and we can then get the proper loading path from that.

Reply:
@keveleigh is working to update our MixedRealityToolkitFiles class to use AssetDatabase APIs. Once that work is complete, this PR will be modified to adapt to the change.